### PR TITLE
Updating Getting Started page 

### PR DESCRIPTION
**Pull request by bromeg**

### CODE REVIEW

File: articles/install-guide/index.md [hunk 1]
```diff
@@ -9,13 +9,44 @@ uid: microsoft.quantum.install
...
+## Choose the environment that's right for you ##
 
-* **[Q# Notebooks](https://github.com/Microsoft/Quantum/tree/master/Samples/src/IntroToIQSharp/Notebook.ipynb)**: explains how to compile and simulate Q# operations inside a Jupyter notebook.
-* **[Teleport](https://github.com/Microsoft/Quantum/tree/master/Samples/src/Teleportation/Notebook.ipynb)**: shows how to implement the teleport algorithm using Q#.
+### Jupyter Notebooks ###
 
-To run these interactively, install IQ# on your machine (see below) and take a look at our [Jupyter notebooks getting started guide](xref:microsoft.quantum.install.jupyter).
+ A great way to get started with Q# before installing the Quantum Development Kit is via some of the Jupyter notebooks available in our [GitHub Quantum repository](https://github.com/Microsoft/Quantum.git).  In particular:
 
+* **[Q# Notebooks](https://github.com/Microsoft/Quantum/tree/master/Samples/src/IntroToIQSharp/Notebook.ipynb)** [(Run online)](https://mybinder.org/v2/gh/Microsoft/Quantum/master?filepath=Samples%2Fsrc%2FIntroToIQSharp%2FNotebook.ipynb) : explains how to compile and simulate Q# operations inside a Jupyter notebook.
```
Comment (cgranade) [inline articles/install-guide/index.md]:
```suggestion
* **[Q# Notebooks](https://github.com/Microsoft/Quantum/tree/master/Samples/src/IntroToIQSharp/Notebook.ipynb)** [(Run online)](https://mybinder.org/v2/gh/Microsoft/Quantum/master?filepath=Samples%2Fsrc%2FIntroToIQSharp%2FNotebook.ipynb): explains how to compile and simulate Q# operations inside a Jupyter notebook.
```

File: articles/install-guide/index.md [hunk 2]
```diff
@@ -9,13 +9,44 @@ uid: microsoft.quantum.install
...
+* **[Teleport](https://github.com/Microsoft/Quantum/tree/master/Samples/src/Teleportation/Notebook.ipynb)** [(Run online)](https://mybinder.org/v2/gh/microsoft/Quantum/master?filepath=Samples%2Fsrc%2FTeleportation%2FNotebook.ipynb): shows how to implement the teleport algorithm using Q#.
+
+To run these locally, install IQ# on your machine (see below) and take a look at our [Jupyter notebooks getting started guide](xref:microsoft.quantum.install.jupyter).
+
+### Visual Studio ###
+
+ If you are already a [Visual Studio](https://visualstudio.microsoft.com/vs/) user, you can install the extension to get started developing Q# in Visual Studio.
```
Comment (cgranade) [inline articles/install-guide/index.md]: Should we clarify 2017 or later here?

File: articles/install-guide/index.md [hunk 3]
```diff
@@ -9,13 +9,44 @@ uid: microsoft.quantum.install
...
+### Python ###
+
+If you are a Python user, check out our [Getting Started with Python and the Quantum Development Kit guide](xref:microsoft.quantum.install.python).
+
+## Learning Quantum Computing with Q# ##
+
+* [Quantum Katas](https://github.com/Microsoft/QuantumKatas)
+* [Samples](https://github.com/Microsoft/Quantum)
+* [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro)
+* [Microsoft Quantum Blog](https://cloudblogs.microsoft.com/quantum/?ext)
```
Comment (cgranade) [inline articles/install-guide/index.md]: Should we link to the devblog here as well? https://devblogs.microsoft.com/qsharp/

File: articles/install-guide/python.md [hunk 4]
```diff
@@ -14,11 +14,11 @@ The [qsharp package for Python](https://pypi.org/project/qsharp/) provides inter
 ## Pre-requisites ##
 
 To get started with the `qsharp` package, you'll need the following prerequisites:
-- IQ#.
-- Python 3.6 or later,
-
-Make sure you have followed the installation steps in the [getting started guide](xref:microsoft.quantum.install) for IQ#.
+- **.NET Core 2.2 or later**,
```
Comment (cgranade) [inline articles/install-guide/python.md]: Users will need the .NET Core SDK in particular (there's also the runtime, but we need the compiler infrastructure that comes with the SDK); would it be good to note that here?

File: articles/install-guide/index.md (hunk 2 above)
Comment (cpalmer2020) [reply articles/install-guide/index.md]: The link points to 2019 and the extension lists 2017 and 2019 as valid.  I don't think we need to call out this here too.

File: articles/install-guide/index.md (hunk 3 above)
Comment (cpalmer2020) [reply articles/install-guide/index.md]: Done

File: articles/install-guide/python.md (hunk 4 above)
Comment (cpalmer2020) [reply articles/install-guide/python.md]: Added "SDK"

File: articles/install-guide/index.md (hunk 3 above)
Comment (bromeg) [reply articles/install-guide/index.md]: oh yeah, I like this as a resource, but I'm not sure what to call it to differentiate from the other quantum blog mentioned. Is Microsoft Quantum Developer Blog or Q# blog sufficient for this?

File: articles/install-guide/index.md (hunk 2 above)
Comment (cgranade) [reply articles/install-guide/index.md]: Sounds good!

**Pull request merged**